test/hasPath.js

```diff
@@ -63,10 +63,6 @@ describe('hasPath', function() {
     eq(R.hasPath(['toString'], bob), false);
   });
 
-  it('returns false for non-objects', function() {
```

Comment:
I removed this test because it uses an empty path, cases for non-objects already have covered by tests below